Source/WebCore/page/Performance.h

```
@@ -34,11 +34,13 @@
 
 #include "ContextDestructionObserver.h"
 #include "DOMHighResTimeStamp.h"
+#include "EventNames.h"
```

I think you should be able to forward declare EventType instead (maybe I'm wrong though) and only import in the CPP

---

Done! Forward declaring `EventType` requires specifying the underlying type, which felt bad at first (ODR violation = UB), but on further reflection the underlying type should be quite stable. Also, our build system detects this ODR violation instead of silently accepting it.